### PR TITLE
Renames ScriptCs.CSharp to ScriptCs.Engine.Roslyn

### PR DESCRIPTION
**Pull request by glennblock**
Fix for #1133

### CODE REVIEW

File: common/CommonVersionInfo.cs [hunk 1]
```diff
@@ -1,9 +1,18 @@
+//------------------------------------------------------------------------------
```
Comment (filipw) [inline common/CommonVersionInfo.cs]: this file should be reverted, the original even says:

```
Do not manually edit this file
```

File: src/ScriptCs.Engine.Roslyn/CSharpModule.cs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿using ScriptCs.Contracts;
 
-namespace ScriptCs.CSharp
+namespace ScriptCs.Engine.Roslyn
 {
     [Module("csharp")]
```
Comment (filipw) [inline src/ScriptCs.Engine.Roslyn/CSharpModule.cs]: the modul;e name should be reverted too

File: src/ScriptCs.Hosting/ModuleLoader.cs [hunk 3]
```diff
@@ -13,7 +13,7 @@ public class ModuleLoader : IModuleLoader
     {
         internal static readonly Dictionary<string, string> DefaultCSharpModules = new Dictionary<string, string>
         {
-            {"csharp", "ScriptCs.CSharp.dll"},
+            {"csharp", "ScriptCs.Engine.Roslyn.dll"},
```
Comment (filipw) [inline src/ScriptCs.Hosting/ModuleLoader.cs]: same here

**Comment by filipw** [comment]
not sure what to do with `scriptcs.visualbasic`?

**Comment by glennblock** [comment]
Hmm I think my changes got overwritten when I pulled Dev. I had renamed the
module already....

**Comment by glennblock** [comment]
@filipw we should remove VB probably. Maybe you should release it as a sep module?

**Comment by filipw** [comment]
yeah let's remove it, can you please update this branch to revert the module rename

**Comment by glennblock** [comment]
@filipw thanks for the feedback and for catching the issues. All issues have been fixed and I removed the VB module.

**Comment by filipw** [comment]
:+1:

**Comment by glennblock** [comment]
@filipw while you are on a roll, how about checking this? https://github.com/scriptcs/scriptcs/pull/1129

**Pull request merged**